src/hotspot/share/cds/metaspaceShared.hpp

```diff
@@ -116,6 +116,10 @@ class MetaspaceShared : AllStatic {
 
   static bool is_shared_dynamic(void* p) NOT_CDS_RETURN_(false);
 
+  static void unrecoverable_loading_error(const char* message);
```

Comment:
This can take a default argument `(const char* message = nullptr)`, so you don't need to pass "" when no extra message is needed.